Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -51,23 +53,46 @@
-
+######## MAY NEED REWORK - COULD ADD KNOWLEDGE OF BRITAIN EVENT TO SPREAD IT OUT BETTER #########
+ BTWK_story_event.0105.t:0 "The Future Council of Dumnonia"
+ BTWK_story_event.0105.desc:0 "To ensure Mordred has the best possible protection, you decide a council to rule the kingdom following your passing will be arranged. A handful of men from your allied kingdoms, along with your loyal vassals, will preside over it. Chiefly among them will be your Bishop, Bedwin. While a Christian, the clergyman is a former soldier of Dumnonia, and has served the kingdom with total loyality to date. \n\n As a final proceeding, you dictate that those who wish to show their undying loyalty to your crown may become sworn protectors of Mordred, bound by death-oaths. One of those men you wish to take the oath is Merlin, a druid and the Lord of Avalon, who is currently missing in his search for the #italic Knowledge of Britain #!. In his stead is one of his protégés - your daughter Morgan. Unfortunately for you, she says he will only accept if Arthur and his men are also made to take the oath."
+ BTWK_story_event.0105.desc_arthur_wed_norwenna:0 ""
+ BTWK_story_event.0105.a:0 "Send word to Arthur, though it grieves me greatly..." 
+ 
+ #BTWK_story_event.0121.t:0 "The Knowledge of Britain"
+ #BTWK_story_event.0121.desc:0 ""
+ #BTWK_story_event.0121.a:0 "Send word to Arthur, though it grieves me greatly..."
+#################################################################################################
+
+ BTWK_story_event.0119.t:0 "An Oath to Mordred"
+ BTWK_story_event.0119.desc_header:0 "The High King is seeking men loyal to Dumnonia to take a death-oath, in the name of protecting Prince Mordred till he is of age to rule.\n\n"
+ BTWK_story_event.0119.desc_vassal:0 "As one of the most powerful Lords within Dumnonia, your liege Uther has offered you the priviledge to swear an oath to ensure the Edling Mordred, Uther's lineage, reaches adulthood safe and sound. Doing so would certainly gain you favours with your current and future liege, given that you live to see the boy ascend to the throne proper. You must take caution with this decision however, as should you fail to uphold your pledge, and Mordred is killed, you may be obliged to seek vengeance against those who caused his death."
+ BTWK_story_event.0119.desc_tewdric:0 "As a long standing friend to Uther, and to Dumnonia, the opportunity has come to cement the alliance between your two realms. By taking the oath, the support of Dumnonia would be a surity if dire circumstances arise in the future. Being a King of a neighbouring realm, you are not expected to protect Mordred as the Dumnonians would. You would, however, always be obliged to be his sword and sheld in times of war, committing your armies to Dumnonia. Consider that this may not always be in your best interests, should Mordred prove not to grow into such an honourable man as Uther."
```

Comment:
1. "As a long standing friend to Uther, and to Dumnonia, the opportunity has come to cement the alliance between your two realms" -> "To us, this request effectively comes as an invitation to renew the alliance between Gwent and Dumnonia after Uther's passing"
2. "By taking the oath, the support of Dumnonia would be a surity if dire circumstances arise in the future." -> "We would be required to swear a death-oath to defend the Edling's life with our own, but this would secure Dumnonia's support against our enemies to the north."
3. "you are not expected to protect Mordred as the Dumnonians would" -> we would not be expected to personally protect Mordred as the other guardians/oath-takers would"